<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="layerselector" name="layerselector_widget">
        <attribute name="title">
         <string>SampleModel1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>+</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="layerplot" name="plot_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>400</height>
         </size>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_plot">
             <property name="text">
              <string>Plot (R)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_plot2">
             <property name="text">
              <string>Plot (RQ^4)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_SA">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Plot (SA)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_fit">
             <property name="text">
              <string>Fit</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="LayerPropertiesWidget" name="layer_properties_widget" native="true">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </widget>
        <widget class="QTextEdit" name="fit_parameters_textEdit">
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>0</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_experiment_data"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_layers"/>
    <addaction name="actionSave_layers"/>
    <addaction name="actionLoad_status"/>
    <addaction name="actionSave_status"/>
    <addaction name="separator"/>
    <addaction name="actionSave_report"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_layers">
   <property name="text">
    <string>Load layers</string>
   </property>
  </action>
  <action name="actionSave_layers">
   <property name="text">
    <string>Save layers</string>
   </property>
  </action>
  <action name="actionLoad_experiment_data">
   <property name="text">
    <string>Experimental data</string>
   </property>
  </action>
  <action name="actionSave_report">
   <property name="text">
    <string>Save report</string>
   </property>
  </action>
  <action name="actionSave_status">
   <property name="text">
    <string>Save state</string>
   </property>
  </action>
  <action name="actionLoad_status">
   <property name="text">
    <string>Load state</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>layerselector</class>
   <extends>QWidget</extends>
   <header location="global">licorne.layerselector</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayerPropertiesWidget</class>
   <extends>QWidget</extends>
   <header location="global">licorne.LayerPropertiesWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>layerplot</class>
   <extends>QWidget</extends>
   <header location="global">licorne.layerplot</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
